README.md

```diff
@@ -42,6 +42,34 @@ In the installation screen, select a minimum of:
 
 All other components are optional.
 
+#### Local DB
+
+Find out if Local DB is already installed
+
+```Batchfile
+powershell -NoProfile sqllocaldb -?
+```
+
+If the command fails or is not existant then Local DB is not installed or needs to be reconfigured. Follow the steps below:
+
+* Download SQL Express [installer](https://www.microsoft.com/en-us/sql-server/sql-server-editions-express)
+* Select the Download Media option
+* Select LocalDB from the option `WHICH PACKAGE WOULD YOU LIKE TO DOWNLOAD`
+* Install `SqlLocalDB.msi` from the download location
+
+#### SQL CMD
```

Comment:
What is the reason for using SQLCMD?

Reply:
With SQLCMD we can automatically provision all the database without requiring the steps of going into SQL Management studio. It is used in the scripts you reviewed :)